src/Dapr.Actors/Reentrancy/ActorLogicalReentrancyContext.cs

```diff
@@ -0,0 +1,36 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Reentrancy
```

Comment:
How many types are in this namespace? It seems unjustified to add this. I'd suggest just putting it in `Dapr.Actors`